my-code/movie-search/src/reducers/reducerChosenMovie.js

```diff
@@ -0,0 +1,13 @@
+import { types } from "../actions";
+
```

Comment:
You can destructure the variable on the top of the function and please use ES6 syntax

```js
export default (state = {}, { type, payload }) => {
```